### PR TITLE
Update README.meson

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: meson_options.txt [hunk 1]
```diff
@@ -46,12 +46,12 @@ option('libblas',
 option('liblapack',
        type : 'string',
        value : 'openblas',
-       description : 'option for BLAS library switching')
+       description : 'option for LAPACK library switching')
```
Comment (dpo) [inline meson_options.txt:49]:
```suggestion
       description : 'LAPACK library against which to link')
```

Comment (dpo) [reply meson_options.txt:49]: Same for BLAS, METIS, and perhaps others.

File: meson_options.txt [hunk 2]
```diff
@@ -111,7 +111,7 @@ option('liblapack_path',
 option('libmetis_path',
        type : 'array',
        value : [],
-       description : 'Additional directories to search METIS 4 library')
+       description : 'Additional directories to search METIS library')
```
Comment (dpo) [inline meson_options.txt:114]:
```suggestion
       description : 'Additional directories to search for the METIS library')
```

**Comment by amontoison** [comment]
Thanks Dominique!

**Pull request merged**